packages/helm-charts/oracle-rbac/templates/role.yaml

```diff
@@ -1,8 +1,12 @@
+{{ range $index, $e := until (.Values.oracleCount | int) }}
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
-  name: {{ template "name" . }}
+  name: {{ template "name" (dict "Values" $.Values "index" $index) }}
 rules:
 - apiGroups: [""]
   resources: ["pods"]
+  resourceNames: ["{{ template "oracle-pod-name" (dict "Values" $.Values "index" $index) }}"]
   verbs: ["get", "update", "patch"]
```

Comment:
I've been meaning to look into this, I *think* only get and patch are needed, could you try running without update and see if things work?